Ruby Basic/task4.rb

```diff
@@ -0,0 +1,21 @@
+puts 'input boolean true or false:'
+
+input_value = gets.chomp.downcase
+
+boolean_keywords = ["true", "false"]
+
+abort('You need enter true/false keyword.') unless boolean_keywords.include? input_value
+
+input_value.include?("true") ? input_value = true : input_value = false
```

Comment:
Assign variable outside of ternary operator is better way of realization. But in this case you already verified incoming parameter using `boolean_keywords.include? input_value`, so now it's enough just reassign variable.
```ruby
  input_value = input_value == 'true'
```

Reply:
done